command/src/main/java/com/jagrosh/jdautilities/command/impl/CommandClientImpl.java

```diff
@@ -822,10 +822,13 @@ private MessageParts getParts(MessageReceivedEvent event) {
             if(rawContent.startsWith("<@"+ event.getJDA().getSelfUser().getId()+">") ||
                     rawContent.startsWith("<@!"+ event.getJDA().getSelfUser().getId()+">")) {
```

Comment:
```suggestion
            if(rawContent.startsWith("<@"+ event.getJDA().getSelfUser().getId()+"> ") || //Ensure that there is a space between the command and the mention
                    rawContent.startsWith("<@!"+ event.getJDA().getSelfUser().getId()+"> ")) {
```

I think only doing this is sufficient.